torch/csrc/jit/passes/quantization.cpp

```diff
@@ -2007,12 +2007,12 @@ void FoldQuantNodesIntoInputsOutputs(std::shared_ptr<Graph>& graph) {
   throw std::runtime_error("Pass not implemented yet!");
 }
 
-void SwapFunctionalLinear(script::Module& module) {
+void SwapFunctionalLinear(Module& module) {
   for (auto& method : module.get_methods()) {
     std::shared_ptr<Graph> g = method.graph();
     SwapFunctionalLinear(g);
   }
-  for (script::Module m : module.children()) {
+  for (Module m : module.children()) {
```

Comment:
```suggestion
  for (Module& m : module.children()) {
```

Reply:
Our convention is that the types we provide as part of the public API are reference types; it's okay to take by value (although also harmless to take by ref)